Aftonbladet.js

```diff
@@ -0,0 +1,311 @@
+{
+	"translatorID": "235c5c47-0c04-4d77-9afe-e22265a017a9",
+	"label": "Aftonbladet",
+	"creator": "Jonatan Svensson Glad",
+	"target": "^https?://((ww(w|c)|tv)\\.)?aftonbladet\\.se",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2016-08-12 01:10:00"
+}
+
+/*
+	Aftonbladet Translator - Parses Aftonbladet articles and creates Zotero-based metadata.
+	Copyright (C) 2016 Jonatan Svensson Glad & Philipp Zumstein
+	This program is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+	This program is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU General Public License for more details.
+	You should have received a copy of the GNU General Public License
+	along with this program. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+	function detectWeb(doc, url) {
+		if (url.indexOf('/article')>-1) {
+			return "newspaperArticle";
+		} else if (getSearchResults(doc, true)) {
+			return "multiple";
+		}
+	}
+	
+	
+	function getSearchResults(doc, checkOnly) {
+		var items = {};
+		var found = false;
+		var rows = ZU.xpath(doc, '//a[contains(@href, "/article")]');
+		for (var i=0; i<rows.length; i++) {
+			var href = rows[i].href;
+			var title = ZU.xpathText(rows[i], './/h2');
+			if (!href || !title) continue;
+			if (checkOnly) return true;
+			found = true;
+			items[href] = title;
+		}
+		return found ? items : false;
+	}
+	
+	
+	function doWeb(doc, url) {
+		if (detectWeb(doc, url) == "multiple") {
+			Zotero.selectItems(getSearchResults(doc, false), function (items) {
+				if (!items) {
+					return true;
+				}
+				var articles = new Array();
+				for (var i in items) {
+					articles.push(i);
+				}
+				ZU.processDocuments(articles, scrape);
+			});
+		} else {
+			scrape(doc, url);
+		}
+	}
+	
+	function scrape(doc) {
```

Comment:
BTW the whole function is indented  too much. If you mark all lines in the function in scaffold you can simple press Shift+Tab to unintend them once.